ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Genesis/Governor.hs

```diff
@@ -297,6 +346,15 @@ densityDisconnect (GenesisWindow sgen) (SecurityParam k) states candidateSuffixe
       -- This matters to ChainSync jumping, where adversarial dynamo and
       -- objector could offer chains of equal density.
       guard $ lb1 >= (if idling0 then lb0 else ub0)
+
+      -- Conjecture 1: All of the guards above imply that peer0 is not aware of
+      -- the best chain up to the last slot of the genesis window anchored at
+      -- the oldest intersection.
+      --
+      -- Conjecture 2: If peer0 is not aware of the best chain up to the last
+      -- slot of the genesis window anchored at the oldest intersection, and
+      -- there is no peer with an older intersection, then all of the guards
+      -- above should be true.
```

Comment:
Other things we might want to mention:

Whether we consider a peer aware of the best chain depends on what headers have been shared by the other peers and whether `peer0` is idle.

Also, we could say "peer0 is not serving the best chain" instead of "peer0 is not aware of the best chain".

Reply:
Edited this to incorporate the substance of the comment.